### DISCUSSION TITLE
Windows 11 the "End Task" Developer Function not working.

### DISCUSSION THREAD

**Discussion by MXRR1S** [valinet/ExplorerPatcher#2680] [Q&A]
In Windows 11, there is a developer function known as "End Task," which introduces a small button to terminate tasks when right-clicking on apps in the Taskbar. This feature is advantageous because it is quicker than navigating to the Task Manager and terminating the process manually. Unfortunately, this functionality is not available when the Taskbar Style is set to Windows 10.

**Comment by MXRR1S** [comment]
When the Style is set to Windows 11 it works without problems.

**Reply by D-CODE5** [reply]
I was just going to ask the same question, I thought it was a Windows 11 problem and no, it's basically what you said. I hope they manage to fix it/implement it with the Windows 10 taskbar style.

**Comment by pyrates999** [comment]
If the developer function is only for windows 11, then the windows 10 code wouldn't have it.  So EP probably can't add it then if the code is not there.

**Reply by MXRR1S** [reply]
The Button is there, it just doesnt do anything. I mean isn't it just the visuals? The code should still be there

**Reply by pyrates999** [reply]
It probably depends on a function in the windows 11 taskbar that doesn't exist in the windows 10 taskbar.

**Reply by MXRR1S** [reply]
do you think its impossible to implement in EP?

**Reply by pyrates999** [reply]
Wait for a developer to respond to find out.

**Comment by D-CODE5** [comment]
I just download the last Pre-release v 22621.3880.66.5, and it works perfectly using windows10(ExplorerPatcher) taskbar style
